### PR TITLE
fix(vendor-core): updating ace editor to fix a typo

### PR DESCRIPTION
**Pull request by MariaAga**
BREAKING CHANGE:
ace editor search import changes, also need to disable loadWorkerFromBlob : 
`import 'brace/ext/searchbox';` -> `import 'ace-builds/src-min-noconflict/ext-searchbox';`
`editorProps={loadWorkerFromBlob: false}`
ISSUES: refs #34062

### CODE REVIEW

File: packages/vendor-core/package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "prop-types": "^15.6.0",
     "rc-input-number": "^6.0.0",
     "react": "^16.9.0",
-    "react-ace": "^6.3.2",
+    "react-ace": "^8.0.0",
```
Comment (sharvit) [inline packages/vendor-core/package.json:69]: Any reason for not upgrading to v9?

**Review by sharvit — COMMENTED** [review]
Thanks for bringing it up @MariaAga

File: packages/vendor-core/package.json (hunk 1 above)
Comment (MariaAga) [reply packages/vendor-core/package.json]: updated to v9

**Comment by adamruzicka** [comment]
@sharvit Anything else to do here?

**Comment by MariaAga** [comment]
Will add some things for https://github.com/theforeman/foreman/pull/8385/files, so it's not ready yet

**Comment by ezr-ondrej** [comment]
And now @MariaAga ? :)

**Comment by MariaAga** [comment]
now it should be ready :)

**Review by ezr-ondrej — APPROVED** [review]
:shipit: Thanks @MariaAga :+1:

**Pull request merged**